flink-ml-lib/src/main/java/org/apache/flink/ml/classification/logisticregression/LogisticRegressionModelData.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.ml.classification.logisticregression;
+
+import org.apache.flink.api.common.serialization.Encoder;
+import org.apache.flink.api.common.typeinfo.TypeInformation;
+import org.apache.flink.configuration.Configuration;
+import org.apache.flink.connector.file.src.reader.SimpleStreamFormat;
+import org.apache.flink.core.fs.FSDataInputStream;
+import org.apache.flink.core.memory.DataInputViewStreamWrapper;
+import org.apache.flink.core.memory.DataOutputViewStreamWrapper;
+import org.apache.flink.ml.linalg.DenseVector;
+import org.apache.flink.ml.linalg.typeinfo.DenseVectorSerializer;
+import org.apache.flink.streaming.api.datastream.DataStream;
+import org.apache.flink.table.api.Table;
+import org.apache.flink.table.api.bridge.java.StreamTableEnvironment;
+import org.apache.flink.table.api.internal.TableImpl;
+
+import java.io.EOFException;
+import java.io.IOException;
+import java.io.OutputStream;
+
+/**
+ * Model data of {@link LogisticRegressionModel}.
+ *
+ * <p>This class also provides methods to convert model data from Table to Datastream, and classes
+ * to save/load model data.
+ */
+public class LogisticRegressionModelData {
+
+    public final DenseVector coefficient;
+
+    public LogisticRegressionModelData(DenseVector coefficient) {
+        this.coefficient = coefficient;
+    }
+
+    /**
+     * Converts the table model to a data stream.
+     *
+     * @param modelData The table model data.
+     * @return The data stream model data.
+     */
+    public static DataStream<LogisticRegressionModelData> getModelDataStream(Table modelData) {
```

Comment:
Good observation here~

In my understanding, the first answer is yes and second answer is no.

As `LogisticRegressionModelData` is not serializable here, Flink is supposed to treat `LogisticRegressionModelData` as a generic and use default serializers (e..g, KryoSerializer) to do the (de)serialization.

If I am correct, we may need to create a type serializer for it or we can simply make it a POJO?

Let's see how @gaoyunhaii thinks.

Reply:
I think @zhipeng93 is right about the current status, it might be easier to make it a POJO. We'd better avoid to use kryo serializer, unless we know the amount of records to transfer is small.

Reply:
I see. Could we make all *ModelData* classes POJO? I think it is possible for model data have large size.

BTW, in order to make sure that we don't accidentally use kryo serializer, how about we set `env.getConfig().disableGenericTypes()` in every algorithm's test?

Reply:
The model data of Kmeans, LR, and NB have been changed to POJOs in the latest commit.

>BTW, in order to make sure that we don't accidentally use kryo serializer, how about we set env.getConfig().disableGenericTypes() in every algorithm's test?

Do you mean that we add this in the unit test?

Reply:
> For small objects, I think we should still allow kryo?

Could you be specific which small object requires kryo? If we do have a reasonable use-case to use kryo, then it is OK not to make this change in unit tests.

Reply:
`TerminateOnMaxIter` accepts a Generic as Input and outputs a Integer. As I see, the input generic is going to use kryo.